phpunit/class-dynamic-blocks-render-test.php

```diff
@@ -38,9 +38,8 @@ function tearDown() {
 
 		$this->dummy_block_instance_number = 0;
 
-		foreach ( WP_Block_Type_Registry::get_instance()->get_all_registered() as $name => $block_type ) {
```

Comment:
While not a behavior we should have wanted to depend on, noting that this change had an unintended consequence of having core server-registered blocks remain registered after the tests complete, which was not the case previously:

https://github.com/WordPress/gutenberg/blob/477e27ea1166bb5e856fa800bc639a25766146c6/phpunit/bootstrap.php#L31-L40

https://github.com/WordPress/gutenberg/blob/477e27ea1166bb5e856fa800bc639a25766146c6/lib/load.php#L24-L27